### PR TITLE
bug 1689241 updating the machineset definition

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1689241

@sunzhaohua2, PTAL?

### CODE REVIEW

File: modules/machineset-creating.adoc [hunk 1]
```diff
@@ -97,16 +97,16 @@ spec:
...
-        sigs.k8s.io/cluster-api-machine-role: <machine_label>
-        sigs.k8s.io/cluster-api-machine-type: <machine_label>
-        sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role:  <machine_label>
+        machine.openshift.io/cluster-api-machine-type: <machine_label>
+        machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
 ...
 ----
 <1> Ensure that both the `sigs.k8s.io/cluster-api-machineset` parameter values
```
Comment (sunzhaohua2) [inline modules/machineset-creating.adoc]: `sigs.k8s.io/cluster-api-machineset` should be `machine.openshift.io/cluster-api-machineset`

**Comment by sunzhaohua2** [comment]
line 169 should be `apiVersion: awsproviderconfig.openshift.io/v1beta1`

File: modules/machineset-creating.adoc [hunk 2]
```diff
@@ -173,8 +173,8 @@ providerSpec:
         volumeSize: 120
         volumeType: gp2
     deviceIndex: 0
```
Comment (sunzhaohua2) [inline modules/machineset-creating.adoc]: the following information need to be added above this line.
```           credentialsSecret:
            name: aws-cloud-credentials
```

**Comment by sunzhaohua2** [comment]
line 190 should change to "<infrastructure_id>-worker-sg"

**Comment by sunzhaohua2** [comment]
line 195 should change to "<infrastructure_id>-private-<AWS-availability-zone>"

**Comment by sunzhaohua2** [comment]
line 197 198 should be deleted

**Comment by sunzhaohua2** [comment]
line 202 should be change to `worker-user-data`

**Comment by sunzhaohua2** [comment]
All `<cluster_name>` should be changed to `<infrastructure_id>`

File: modules/machineset-creating.adoc [hunk 3]
```diff
@@ -48,17 +48,17 @@ metadata:
...
-    sigs.k8s.io/cluster-api-cluster: <cluster_name> <2>
-    sigs.k8s.io/cluster-api-machine-role: <machine_label> <3>
-    sigs.k8s.io/cluster-api-machine-type: <machine_label> <3>
+    machine.openshift.io/cluster-api-cluster:  <infrastructure_id> <2>
+    machine.openshift.io/cluster-api-machine-role: <machine_label> <3>
+    machine.openshift.io/cluster-api-machine-type: <machine_label> <3>
   name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <4>
   namespace: openshift-machine-api
   resourceVersion: "9249" <1>
   selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-machine-api/machinesets/<cluster_name>-<machine_label>-<AWS-availability-zone> <1>
```
Comment (sunzhaohua2) [inline modules/machineset-creating.adoc]: should be `/apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machinesets/ <infrastructure_id>-<machine_label>-<AWS-availability-zone> `

**Comment by kalexand-rh** [comment]
@sunzhaohua2, I've made the changes that you requested. Will you please confirm?

**Comment by sunzhaohua2** [comment]
LGTM, thank you!

**Comment by kalexand-rh** [comment]
Thanks! Merging.

File: modules/machineset-creating.adoc [hunk 4]
```diff
@@ -48,17 +48,21 @@ metadata:
   creationTimestamp: 2019-02-15T16:32:56Z <1>
   generation: 1 <1>
   labels:
+<<<<<<< HEAD
     machine.openshift.io/cluster-api-cluster: <cluster_name> <2>
+=======
+    machine.openshift.io/cluster-api-cluster:  <infrastructure_id> <2>
+>>>>>>> bug 1689241 updating the machineset definition
```
Comment (wking) [inline modules/machineset-creating.adoc]: Are these conflict markers intentional, or did they sneak in during a rebase?

Comment (kalexand-rh) [reply modules/machineset-creating.adoc]: Thank you! https://github.com/openshift/openshift-docs/pull/14593

**Pull request merged**